src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,350 @@
+            this.setStickyColumnPositions();
+            this.applyStickyClassesToCells();
+        }
+    }
+
+    setStickyColumnPositions() {
+        const {
+            stickyColumnCount,
+            definition,
+            basic,
+            fullWidth,
+            singleLine,
+            celled,
+            fixed,
+        } = this.props;
+
+        if (stickyColumnCount > 0 && this.tableRef) {
+            const stickyCells = this.tableRef.querySelectorAll(`.table--cell:nth-child(-n+${stickyColumnCount})`);
+            let cellWidths = 0;
+            let cellCount = 0;
+            /**
+             * Reducing each left sticky column position will help to not have a separation
+             * between them, so the content behind won't be visible when scrolling
+             */
+            let leftOverFloatSpace = 1;
+            let auxLeftOverFloatSpace = 0;
+            if (basic) {
+                leftOverFloatSpace = 4;
+                auxLeftOverFloatSpace = -1;
+                if (celled) {
+                    leftOverFloatSpace = 4;
+                    auxLeftOverFloatSpace = -1;
+                }
+                if (fixed || singleLine) {
+                    leftOverFloatSpace = 4;
+                    auxLeftOverFloatSpace = 1;
+                }
+            } else {
+                leftOverFloatSpace = 2;
+                auxLeftOverFloatSpace = 1;
+                if (celled) {
+                    auxLeftOverFloatSpace = 2;
+                }
+                if (fixed || singleLine) {
+                    leftOverFloatSpace = 3;
+                    auxLeftOverFloatSpace = 2;
+                }
+            }
+
+            for (let rootIndex = 0; rootIndex < stickyCells.length; rootIndex += 1) {
+                cellCount += 1;
+                const shouldChangeInitialDefinitionCell = definition &&
+                    !fullWidth && rootIndex === 0;
+                if (shouldChangeInitialDefinitionCell) {
+                    stickyCells[rootIndex].style.backgroundColor = backgroundColor.default;
+                }
+                if (cellCount <= stickyColumnCount && cellCount > 1) {
+                    cellWidths += stickyCells[rootIndex - 1].clientWidth;
+                    /**
+                     * This makes definition/fullwidth work
+                     * otherwise we'd have big separation space between sticky columns
+                     */
+                    if (definition && !basic) {
+                        const isFirstHeaderColumn = rootIndex > 0 && cellCount > 0 && stickyCells[rootIndex].className.includes('header');
+                        const definitionLeftOverflow = isFirstHeaderColumn ? 4 : 3;
+                        stickyCells[rootIndex].style.left = `${cellWidths - definitionLeftOverflow}px`;
+                    } else {
+                        // eslint-disable-next-line max-len
```

Comment:
Please remove `eslint-disable-next-line max-len` and fix in code. We need to try our best to not go to the disable eslint comments if we are able to easily.

Reply:
done!